test/e2e/specs/editor/blocks/navigation.spec.js

```diff
@@ -92,13 +90,34 @@ test.describe(
 			] );
 			await page.locator( 'role=button[name="Close panel"i]' ).click();
 
-			//check the block in the frontend.
-			await page.goto( '/' );
-			await expect(
-				page.locator(
-					'role=navigation >> role=link[name="WordPress"i]'
-				)
-			).toBeVisible();
+			// Check the block in the frontend.
+			await navBlockUtils.selectNavigationItemOnFrontend( 'WordPress' );
+		} );
+
+		test( 'default to the only existing classic menu if there are no block menus', async ( {
+			admin,
+			editor,
+			requestUtils,
+			navBlockUtils,
+		} ) => {
+			// Create a classic menu.
+			await requestUtils.createClassicMenu( 'Test Classic 1' );
+			await admin.createNewPost();
+
+			await editor.insertBlock( { name: 'core/navigation' } );
+			// We need to check the canvas after inserting the navigation block to be able to target the block.
+			const navigationBlock = editor.canvas.locator(
+				'role=document[name="Block: Navigation"i]'
+			);
+			await expect( navigationBlock ).toBeVisible();
```

Comment:
```suggestion
			await expect.poll( editor.getBlocks ).toMatchObject( [ {
				name: 'core/navigation',
			} ] );

```

Not sure if this is better!

Reply:
yeah that works!